lib/foreman_bootdisk/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ForemanBootdisk
-  VERSION = '16.1.0'
+  VERSION = '16.0.0'
```

Comment:
Do not touch this line in this PR please. There is a conflict by the way because of this.

Reply:
Fixed !